Add XTerraMetrics Provider

Akari415 opened the pull request:
@RareData Can you take a look at our entry? You can reach me on Telegram at @Akari415.

Akari415 commented:
@RareData Just giving this a bump. Should have everything in the files working correctly. Trying to get in on V2 as early as possible. Let me know if there are any issues.

RareData commented:
Make sure your images are way below 24 KB. Thanks

RareData commented:
Also delete the all lowercase image: assets/0xb0840104e84da02254b043fd4c205415e759bf1e.png

Akari415 commented:
@RareData I have changed the image to a more optimized version at 20mbs now from 23mb. That is the best we can do for our current logo. Please let me know if this is adequate otherwise me and my team may need to revise our current logo.

Also deleted the incorrect png asset that had improper capitalization. Thanks.

Akari415 commented:
@RareData Me and my team have revised the logo and made it down to a 7KB size now.

Akari415 commented:
Hey @RareData, not sure if this is the right channels to request help with but hopefully you can help me with a question. Now that this pull request was accepted and merged. I'm not seeing our signal provider address/name able to be searched in the Bifrost Wallet app. Is there an issue with our merged files/completed prerequisites or is there something that anyone on the github can help us with? Please let me know if you can help or if there is a better place to ask for my issue?
